addons/l10n_es_edi_facturae/data/facturae_templates.xml

```diff
@@ -0,0 +1,329 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <!-- Sub-template used for every instance of AddressType -->
+        <template id="address_type">
+            <t t-if="partner_country_code == 'ESP'">
+                <AddressInSpain>
+                    <Address t-out="', '.join(val for val in (partner.street, partner.street2) if val)"/>
+                    <PostCode t-out="str(partner.zip)"/>
+                    <Town t-out="partner.city"/>
+                    <Province t-out="partner.state_id.name if partner.state_id else partner.country_id.name"/>
+                    <CountryCode t-out="partner_country_code"/>
+                </AddressInSpain>
+            </t>
+            <t t-else="">
+                <OverseasAddress>
+                    <Address t-out="', '.join(val for val in (partner.street, partner.street2) if val)"/>
+                    <PostCodeAndTown t-out="' '.join(var for var in (str(partner.zip), partner.city) if var)"/>
+                    <Province t-out="partner.state_id.name if partner.state_id else partner.country_id.name"/>
+                    <CountryCode t-out="partner_country_code"/>
+                </OverseasAddress>
+            </t>
+        </template>
+
+        <!-- Sub-template used for every instance of ContactDetails -->
+        <template id="contact_details_type">
+            <ContactDetails>
+                <Telephone t-out="partner_phone"/>
+                <WebAddress t-out="partner.website"/>
+                <ElectronicMail t-out="partner.email"/>
+            </ContactDetails>
+        </template>
+
+        <!-- Sub-template used for every instance of AccountType -->
+        <template id="account_type">
+            <IBAN t-out="account['IBAN']"/>
+            <AccountNumber t-out="account['AccountNumber']"/>
+            <BankCode t-out="account.get('BankCode')"/>
+            <BranchCode t-out="account.get('BranchCode')"/>
+            <t t-call="l10n_es_edi_facturae.address_type"><t t-set="address" t-value="account['Address']"/></t>
+            <BIC t-out="account.get('BIC')"/>
+        </template>
+
+        <!-- Sub-template used for every instance of InstallmentType -->
+        <template id="installment_type">
```

Comment:
We should remove this if we do not use it.  (or calculate it from the reconciliations, but it is a bit late to change that now quickly)